src/parsemultipart.jl

```diff
@@ -0,0 +1,52 @@
+function parse_multipart_chunk!(chunk, d)
+    re = r"(\n){2,}|(\r){2,}|[\r\n]{4,}"
```

Comment:
It might be a little clearer to put all the regexes as top-level `const BOUNDARY_REGEX = r"(\n){2,}...`, so it's clear they're const and they have more descriptive names.